core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -213,6 +213,7 @@ var StoryEditorPage = function() {
   };
 
   this.navigateToStoryEditorTab = async function() {
+    await waitFor.pageToFullyLoad();
```

Comment:
Hi @EricZLou, I pushed a commit to fix the e2e error. I was able to reproduce it locally and this line seemed to fix the error.

The error I kept facing initially was this:

```
1) Chapter editor functionality should create a basic chapter with a thumbnail.
  - Failed: element click intercepted: Element <span class="story-name protractor-test-back-to-story-editor-button ng-binding" ng-click="$ctrl.navigateToStoryEditor()" role="button" tabindex="0">...</span> is not clickable at point (80, 8). Other element would receive the click: <img ng-src="/assets/images/logo/288x128_logo_white.png" class="oppia-logo" alt="Oppia Home" ng-style="!$ctrl.windowIsNarrow &amp;&amp; {'width': '90px'}" src="/assets/images/logo/288x128_logo_white.png" style="width: 90px;">
    (Session info: chrome=90.0.4430.93)
    (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.8.0-53-generic x86_64)
```

Looking at the screen where it occured, it seemed like the test was trying to find and click on the "back to story" button.
![backtostory](https://user-images.githubusercontent.com/11008603/120007219-4b888e80-bff7-11eb-8c6f-d5d98f3f37ca.png)


Based on screenshot in the artifacts (+ watching the local runs), it seems like this was hidden beneath the top nav bar.

![Chapter editor functionality should create a basic chapter with a thumbnail](https://user-images.githubusercontent.com/11008603/120007464-90142a00-bff7-11eb-90a1-4cfce05cb449.png)

Protractor could find the element but could not click on it because it was beneath the nav bar (error would be different if it could not find the element).

We do a `general.scrollToTop()` before clicking on the `backToStoryEditorButton` to make sure that the button is visible before clicking on it. However, in one scenario we seem to do a refresh before the "navigateToStoryEditorTab` operation (see [code](https://github.com/oppia/oppia/blob/32ec4673f942bedf0b6227c58bbbd3815ae1f2f5/core/tests/protractor_desktop/topicAndStoryEditorFileUploadFeatures.js#L243-L246)), this might be interfering with the scroll to top operation. So with the `waitFor.pageToFullyLoad()` now, we wait for the page to properly load before naviating to the story editor tab. I tested this locally and seemed to work fine.

Reply:
Wow thanks so much! I've been trying to debug this and was too focused on figuring out how my RTE changes were making the test fail. So I wasn't liking in the right places. But seems like the failure wasn't caused by my RTE changes...

Reply:
Yeah that's weird actually, I still don't understand why we are seeing this only now. It does not look related to your changes AFAICT.